rclcpp/include/rclcpp/node.hpp

```diff
@@ -320,6 +321,21 @@ class Node : public std::enable_shared_from_this<Node>
     const rclcpp::QoS & qos = rclcpp::ServicesQoS(),
     rclcpp::CallbackGroup::SharedPtr group = nullptr);
 
+  /// Create and return a GenericClient.
+  /**
+   * \param[in] service_name The name on which the service is accessible.
+   * \param[in] service_type The name of service type, e.g. "test_msgs/srv/BasicTypes"
```

Comment:
as an example for doc section, it would be better to be actual standard service type? instead of test service types?

```suggestion
   * \param[in] service_type The name of service type, e.g. "std_srvs/srv/SetBool"
```

Reply:
Yes. Update it.